### DISCUSSION TITLE
Column Text Search Boxes

### DISCUSSION THREAD

**Discussion by ILSua** [microsoft-search/pnp-modern-search#3216] [Q&A]
Hello all,

In on-prem search, as with PnP Search V3, there was the ability to have a text search box on each column header.
I am wondering if this is still possible to configure, or if this is planned for any further versions?

Thank you all.

NB: Images are from on-prem search, but presented similar in PnP Search V3.
![image 2](https://github.com/microsoft-search/pnp-modern-search/assets/123122451/1a57035b-1b31-47fa-a4ab-c4bbd685b639)
![image1](https://github.com/microsoft-search/pnp-modern-search/assets/123122451/857c9864-5226-4277-ae42-dbd385583949)

**Comment by kasperbolarsen** [comment]
I am afraid that the answer to both questions is a No, not possible and AFAIK no plans about adding it.
